src/com/company/Zadania.java

```diff
@@ -1332,5 +1332,29 @@ public void zadanie43() {
             System.out.print(x + ",");
         }
     }
+
+    //Write a program, which will find all such numbers between 1000 and 3000 (both included)
+    // such that each digit of the number is an even number.
+    //The numbers obtained should be printed in a comma-separated sequence on a single line.
+
+    public void zadanie44() {
+        int helper;
+        int digit;
+        int counter = 0;
+        for (int i = 1000; i < 3001; i++) {
+            helper = i;
+            for (int j = 0; j < 4; j++) {
+                digit = helper % 10;
+                if (digit % 2 == 0) {
+                    counter++;
+                }
+                helper = helper / 10;
```

Comment:
helper /= 10,
rownie dobrze to za j++ po przecinku mozesz dac
for (int j=0; j<4; ++j, helper/=10)
powinno sie skompilowac, ale czy ladniej , nie wiem ;]